.circleci/config.yml

```diff
@@ -142,7 +142,7 @@ commands:
             echo ${PARITY_VERSION} > /tmp/parity-version
             echo ${SOLC_VERSION} > /tmp/solc-version
       - restore_cache:
-          key: system-deps-v7-{{ arch }}-geth-{{ checksum "/tmp/geth-version" }}-parity-{{ checksum "/tmp/parity-version" }}-solc-{{ checksum "/tmp/solc-version" }}
+          key: system-deps-v8-{{ arch }}-geth-{{ checksum "/tmp/geth-version" }}-parity-{{ checksum "/tmp/parity-version" }}-solc-{{ checksum "/tmp/solc-version" }}
```

Comment:
The checksums should automatically cause a new cache to be used.
It shouldn't be necessary to increase the version manually, that's only there as a fallback when the cache gets corrupted for some reason

Reply:
I accidentally changes the solc link, which messed up the cache. A new checksum would have only been created if I changed one of the software version numbers, but all of these were already correct, so I increased the `v` value in the key.